docs/root/modules/gsuite/config.md

```diff
@@ -15,3 +15,77 @@ Follow these steps to analyze GSuite users and groups with Cartography.
     1.  Export the environmental variables:
         1. `GSUITE_GOOGLE_APPLICATION_CREDENTIALS` - location of the credentials file.
         1. `GSUITE_DELEGATED_ADMIN` - email address that you created in step 2
+
+1. Using Oauth
+
+    1. Create an App on [Google Cloud Console](https://console.cloud.google.com/)
+    1. Refer to follow documentation if needed:
+        1. https://developers.google.com/admin-sdk/directory/v1/quickstart/python
+        1. https://developers.google.com/workspace/guides/get-started
+        1. https://support.google.com/a/answer/7281227?hl=fr
+    1. Download credentials file
+    1. Use helper script for OAuth flow to obtain refresh_token
+    1. Serialize needed secret
```

Comment:
Let's either add a step here telling the user to delete the credentials.json file that was saved to disk, or add a step in the script below that deletes the file.